audit/audit-gcp.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+# set -x -e
+CNCF_GCP_ORG=758905017065
```

Comment:
You could source `infra/gcp/lib.sh` if there are any more utilities to re-use.  This is unlikely to change :)